section2/exercises/methods.rb

```diff
@@ -11,21 +11,38 @@ def print_name
 print_name
 
 # Write a method that takes a name as an argument and prints it:
-def print_name(name)
-  # YOUR CODE HERE
+def print_name
+  p "Albus Dumbledore"
 end
 
-print_name("Albus Dumbledore")
+print_name
```

Comment:
You deleted the parameter `name` and hardcoded a value instead of using the argument within the method definition. How would you fix this?